runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/PTransformTranslation.java

```diff
@@ -101,6 +101,8 @@
       "beam:transform:combine_per_key_merge_accumulators:v1";
   public static final String COMBINE_PER_KEY_EXTRACT_OUTPUTS_TRANSFORM_URN =
       "beam:transform:combine_per_key_extract_outputs:v1";
+  public static final String COMBINE_PER_KEY_CONVERT_TO_ACCUMULATORS_TRANSFORM_URN =
```

Comment:
Please add to the checkState in the static initialization block to make sure this value doesn't deviate from the string constant here.

Note, we do it this way so the Java compiler can treat the value as a string constant.

Reply:
Done!